src/ostree/ot-builtin-commit.c

```diff
@@ -686,6 +686,18 @@ ostree_builtin_commit (int argc, char **argv, OstreeCommandInvocation *invocatio
       tree_type = g_strndup (tree, eq - tree);
       tree = eq + 1;
 
+      if (first && opt_selinux_policy)
```

Comment:
I think this could even go outside of the for-tree-iter body, the `first` condition is effectively constraining it to only run once.

Reply:
Hmm true but it was a bit more subtle than that.  Updated :new: